plugins/polio/js/src/pages/IM/utils.ts

```diff
@@ -224,10 +224,11 @@ export const getLqasImMapLayer = ({
             status: shape.data ? determineStatus(shape.data) : IN_SCOPE,
         }));
     }
-    return shapesWithData
-        .filter(shape => Boolean(shape.data))
-        .map(shape => ({
-            ...shape,
-            status: determineStatus(shape.data),
-        }));
+    // return shapesWithData
```

Comment:
Is this normal?

Reply:
Yes, I return en empty list (i.e: no  districts shapes) when there is no scope, i.o all the district shapes for the country  (like the commented out code did)